### PR TITLE
Add  Task1.txt

### PR DESCRIPTION
**Pull request by BohdanaMarysh**

### CODE REVIEW

File: Task1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Test
```
Comment (Hmeliumm) [inline Task1.txt:1]: Please rewrite to more logical text (at least Line 1 or something else)

Comment (Hmeliumm) [inline Task1.txt:1]: Please rewrite to more logical text (at least Line 1 or something else)

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: Task1.txt (hunk 1 above)
Comment (BohdanaMarysh) [reply Task1.txt:1]: Fixed
Thank you for your comment

Comment (Hmeliumm) [reply Task1.txt:1]: Have you added a commit?

Comment (BohdanaMarysh) [reply Task1.txt]: Yes, i did

**Review by Hmeliumm — APPROVED** [review]

**Pull request merged**